src/guides/v2.3/ui_comp_guide/components/wysiwyg/configure-tinymce-editor.md

```diff
@@ -56,6 +56,13 @@ The `DefaultConfigProvider` class returns the data required to render the TinyMC
 From the example, the class `Magento\PageBuilder\Model\Wysiwyg\DefaultConfigProvider` overrides the existing configuration provided by the `Magento_CMS` module.
 
 ## Extending the TinyMCE editor
+When extending the TinyMCE editor it is necessary to add the PageBuilder module to the load order of the custom module by adding the following to the `module.xml` file.
```

Comment:
Blank line between header and content.

Reply:
Wonder if the linter should have caught that.

Reply:
Sorry I'm not seeing what you mean. Do you mean line 60?

All other sections have a blank line between content and ```xml

Reply:
Line 59 should be a blank line. "When extenting..." is line 60 and then yes, another blank line between the the content and the code block.

Reply:
Ah, you mean ADD a blank line 😄 
Gotta be more precise guv 😉